pkg/gstatus/gstatus.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 ScyllaDB
```

Comment:
Better to use full word `global` in the file name, IMHO.
For the moment just word `g` is not self-explanatory.